Automattic/woocommerce-payments
# Format customer currency based on locale

Fixes #2247 

#### Changes proposed in this Pull Request

Instead of formatting the currency by the locale's country, use the full locale with the data introduced in #2357.









#### Testing instructions





* Set your store default currency to USD
* Enable EUR in your store under WooCommerce > Settings > Multi-Currency
* View the shop page and switch from USD to EUR
* The formatting should (still) change from $1,000.00 to 1.000,00 €
* Modify the user's locale to `nl_NL` and view the shop page
* The formatting should be € 1.000,00
* Tests should pass

PS: The EUR formatting will use ` ` instead of `.` as the thousand separator, but this will be fixed in a separate PR.

-------------------
